content/doc/book/installing/_jenkins-command-parameters.adoc

```diff
@@ -139,7 +139,7 @@ the keystore to PEM] and use following command line options:
 
 === Using HTTP/2
 
-With Java 8 will not use HTTP/2 unless alpn boot jar is included in the bootclasspath.
+Java 8 will not use HTTP/2 unless alpn boot jar is included in the bootclasspath.
```

Comment:
this sentence isn't true anymore btw, https://github.com/jenkinsci/winstone#http2-support

Reply:
Thanks very much for the pointer to the winstone docs on HTTP/2 support.  

I'm not confident that I've understood what is and is not required for HTTP/2 support with Jetty.  I think the instructions should be (skipping Java 9 and 10, since they are not supported by Jenkins):

== Java 8u252 and Java 11+

* Download the [jetty-alpn-java-server](https://mvnrepository.com/artifact/org.eclipse.jetty/jetty-alpn-java-server) with the version number matching the Jetty version bundled with your Jenkins version
* Place the jetty-alpn-java-server.jar file in a directory accessible to the JVM
* Add the `--extraLibFolder=/path/to/extra/lib/folder` to the Java command line arguments used to start Jenkins

== Java 8u242 and earlier

* Download the [jetty-alpn-java-server](https://mvnrepository.com/artifact/org.eclipse.jetty/jetty-alpn-java-server) with the version number matching the Jetty version bundled with your Jenkins version
* Place the jetty-alpn-java-server.jar file in a directory accessible to the JVM
* Add the jetty-alpn-java-server.jar to the JVM boot classpath by adding `-Xbootclasspath/p:/path/to/jetty-alpn-java-server.jar` to the Java command line arguments used to start Jenkins

Is that a correct expression of what I read at https://github.com/jenkinsci/winstone#http2-support ?

CC: @olamy

Reply:
Looks right to me

Reply:
Thanks!  Could you (and preferably @olamy ) review 2ac002c98ad118f9ae382f5efa4124d6039b88fc ?

Reply:
@olamy the changes have been made in the latest commits.  I believe I've covered all the issues you identified.  I have **not** performed an end to end test to confirm the instructions are correct.